src/nano/version.py

```diff
@@ -0,0 +1 @@
+__version__ = '2.1.0'
```

Comment:
Please put this back `nano/version.py`

Reply:
I had changed in setup.py, bumping version should have been working for the path /src/nano/version.py

`with open(os.path.join(here, 'src', NAME, 'version.py'), 'rb') as f:`